be/src/olap/olap_common.h

```diff
@@ -241,7 +246,83 @@ typedef std::set<uint32_t> UniqueIdSet;
 // Column unique Id -> column id map
 typedef std::map<ColumnId, ColumnId> UniqueIdToColumnIdMap;
 
-typedef int64_t RowsetId;
+// 8 bit, id version
+// 120 bit, backend random number generated by uuid
+// 64 bit, inc number from 0
+struct RowsetId {
+    int8_t version = 0;
+    int64_t hi = 0;
+    int64_t mi = 0;
+    int64_t lo = 0;
```

Comment:
uint64_t?

Reply:
UniqueId generator using int64_t so that I use int64_t here